### ISSUE TITLE
Will you be updating/compiling the current version into English?

### ISSUE THREAD

**Issue by tBal-WithaTouchofMalice** [Rainbell129/Obsidian-Homepage#31]
I have been an Obsidian user for a short while, and I absolutely love this vault - thank-you so much for sharing it.

Please pardon my use of English as it is my best option out of the 2 languages that I know.

I am not a coder, but I am learning my way around, so please forgive me if this is the wrong place to ask, or if it is a stupid question.

The latest version updated last week does not have the ` snippets` folder in the `.obsidian` folder included.  The regular version does have it included, and all is fine except for the 'snippets' folder (which may or may not work in the English version.

Edit: to clarify that the `snippets `folder is missing in the `.obsidian` folder in the Rainbell English Folder

**Comment by DhananjayPorwal** [comment]
Hi, this is the right place to ask. I agree with your point about the English part. Additionally, the author clearly mentioned that we need to use the `Rainbell English` folder as a vault, which contains `.obsidian`.

**Comment by tBal-WithaTouchofMalice** [comment]
@DhananjayPorwal 
Yes, I used Rainbell English, but that `.obsidian` folder does not contain the `snippets` folder.

I was incorrect when I typed `.obsidian` instead of `snippets `when I created this issue.  I have edited the original and added the edit details.

**Comment by Rainbell129** [comment]
snippets are in .obsidian
在 2024年9月10日 +0800 02:19，Nunyfu kin biznatch ***@***.***>，写道：

@DhananjayPorwal<https://github.com/DhananjayPorwal>
Yes, I used Rainbell English, but that .obsidian folder does not contain the snippets folder.

I was incorrect when I typed '.obsidian' instead of 'snippets' when I created this issue.
